packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -175,18 +254,17 @@ class DateUtil {
    * @return {string} - The formatted date string.
    */
   static formatMomentDate(momentDate, format) {
-    return momentDate && momentDate.isValid() ? momentDate.format(format, true) : undefined;
+    return momentDate?.isValid() ? momentDate.format(format, true) : undefined;
   }
 
   /**
    * Determines if a provided date input value is valid.
-   * Valid inputs are either empty strings or contain only numeric, `/`, and '.' characters.
+   * Valid inputs are either empty strings or contain only numeric characters.
    * @param {String} value Value to validate
    * @return True if the value is valid, false otherwise.
    */
   static validDateInput(value) {
-    /* eslint-disable-next-line no-useless-escape */
-    return value.length === 0 || /^[\d\/.-]+$/.test(value);
+    return value.length === 0 || /^\d+$/.test(value);
```

Comment:
Can we add some jest tests to validate this change? I didn't find any existing ones.

Reply:
No sure if this is sufficient, but there is a wdio test which checks for the key limitations: https://github.com/cerner/terra-framework/blob/ba9d3c9b977ded21c1db0789cbe2b1e43dd3c7c6/packages/terra-date-picker/tests/wdio/date-picker-spec.js#L349-L361